### PR TITLE
Token display with Terra Assets rather than static data + various UI improvements.

### PR DESCRIPTION
**Pull request by simke9445**
Changes include:
- Extended following query hooks with tokenDisplay metadata from `https://assets.terra.money/cw20/tokens.json`:
  - useBorrowMarketQuery
    - overseerWhitelist elements extended with tokenDisplay, joined by `collateral_token`.
  - useBAssetInfoQuery
  - useBAssetInfoListQuery
  - useBAssetInfoByTokenAddrQuery
  - useBAssetInfoByTokenSymbolQuery
    - all of the above extended with tokenDisplay, joined by `converterConfig.anchor_token_address` and `converterConfig.wormhole_token_address` respectively.
    - due to a known issue of `useQuery` always returning a new object, silenced the exhaustive-deps warning explicitly, as logic has no issues. reference: https://github.com/tannerlinsley/react-query/issues/1858 
- All of the above is encapsulated within `useQueryWithTokenDisplay` hook.
- Updated `TokenIcon` to accept tokenDisplay - existing behavior multiplexed.
- Removed prettify helper functions and static code across the app.
- Display 0 `bETH` balance on `WhImport` page, putting it in line with `WhExport` and `webEth` display.
image

This PR includes various UI improvements.
- Split bAsset rewards by bAsset
image
- Improvements to transaction screens
  - claim airdrop - added more tx info
  - withdraw - refetch history on tx progress change
  - send - display proper asset denominator (i.e. bETH rather than UST)
- Minor improvements (reordering, styles, etc.)

### CODE REVIEW

**Comment by cosullivan** [comment]
@simke9445 with the cw20TokenDisplayInfosQuery function, we need to be able to ensure that we can support assets on the testnet. 

I'm not 100% sure how these work though, ie, we get people to create PR's and the assets are in the Github repo under a "testnet" property, but not sure how these will show up on assets.terra.money.

Can you ensure that is going to work and maybe ping Sim and he might be able to guide you through how it works.

**Comment by simke9445** [comment]
> @simke9445 with the cw20TokenDisplayInfosQuery function, we need to be able to ensure that we can support assets on the testnet.
> 
> I'm not 100% sure how these work though, ie, we get people to create PR's and the assets are in the Github repo under a "testnet" property, but not sure how these will show up on assets.terra.money.
> 
> Can you ensure that is going to work and maybe ping Sim and he might be able to guide you through how it works.

They are, after a pull request is merged to the assets repo, the CD pipeline will deploy the respective folders under `assets.terra.money` (confirmed with Sim too).

Example of previous `testnet` merge requests, which are present at `https://assets.terra.money/cw20/tokens.json`:
- https://github.com/terra-money/assets/pull/238/files 

I've also verified e2e flows via `testnet`, and the icons are properly rendered.

File: app/src/@anchor-protocol/app-fns/forms/borrow/borrow.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { OverseerWhitelistWithDisplay } from '@anchor-protocol/app-provider';
+import { OverseerWhitelistWithDisplay } from '@anchor-protocol/app-provider/queries/borrow/utils/tokenDisplay';
```
Comment (cosullivan) [inline app/src/@anchor-protocol/app-fns/forms/borrow/borrow.ts:1]: export these types from the index.ts in the app-provider folder so we can avoid the extended imports.

Comment (simke9445) [reply app/src/@anchor-protocol/app-fns/forms/borrow/borrow.ts]: done

**Review by cosullivan — APPROVED** [review]

**Pull request merged**